pkg/crc/machine/kubeconfig.go

```diff
@@ -53,6 +53,14 @@ func isReady(status *cluster.Status) bool {
 	return status.Available && !status.Progressing && !status.Degraded && !status.Disabled
 }
 
+func GetClusterCACert(kubeconfigFile string) (string, error) {
```

Comment:
This function looks like `certificateAuthority` few lines below. Can we merge both ?

Reply:
@guillaumerose right, done :+1: